<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>538</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>435</width>
    <height>538</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>435</width>
     <height>538</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>435</width>
     <height>538</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: #1B262C;
}

QLineEdit{
background-color: rgb(18, 26, 30);
color: white;
border: 2px solid gray;
border-radius: 5px;}

QLineEdit:hover{
border: 2px solid rgb(174, 173, 172);
}

QLineEdit:focus{
border: 2px solid rgb(7,134,193)}</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>471</width>
      <height>531</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLineEdit" name="mainEmail">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mainPassword">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="loginLbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: red
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loginBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>10</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
 background-color: #0F4C75;
 color: darkgray;
 border-style: outset;
 border-width: 2px;
 border-radius: 10px;
 border-color: black;
 min-width: 10em;
 padding: 6px;}

QPushButton:hover{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:focusr{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:pressed {
 background-color: #0F4C75;
 border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createAccountBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>10</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>146</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
 background-color: #3282B8;
 color: rgb(228,240,237);
 border-style: outset;
 border-width: 2px;
 border-radius: 10px;
 border-color: black;
 min-width: 10em;
 padding: 6px;}

QPushButton:hover{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:focus{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:pressed {
 background-color: #0F4C75;
 border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Create Account</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="forgotPasswordBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>10</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>146</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
 background-color: #3282B8;
 color: rgb(228,240,237);
 border-style: outset;
 border-width: 2px;
 border-radius: 10px;
 border-color: black;
 min-width: 10em;
 padding: 6px;}

QPushButton:hover{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:focus{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:pressed {
 background-color: #0F4C75;
 border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Forgot Password?</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
